src/main/java/frc/robot/RobotContainer.java

```diff
@@ -19,6 +21,17 @@ public class RobotContainer {
     private DoubleSupplier rightDistanceSupplier;
     private Supplier<Rotation2d> gyroSupplier;
     private Supplier<Pose2d> simulatedPoseSupplier;
+    private ShooterIntakeSubsystem intakeSubsystem =
+            new ShooterIntakeSubsystem(
+                    new ShooterIntakeIO() {
+
+                        @Override
+                        public void setVoltage(double volts) {
+                            // TODO Auto-generated method stub
+                            throw new UnsupportedOperationException(
+                                    "Unimplemented method 'setVoltage'");
+                        }
+                    });
```

Comment:
We should be implementing the IO classes in separate files - one for hardware and one for sim. They both should use the interface you've created.

Reply:
Forgot to push new commit

Reply:
I just forgot to remove those lines of code I believe.